### ISSUE TITLE
Web Analytics and Solid principle

### ISSUE THREAD

**Issue by Ryuno-Ki** [solid/information#76]
This is something which came to my mind after reading https://github.com/solid/node-solid-server/issues/1153

Please point me to the correct place if I'm wrong here (would like to discuss „best practice”).

Now to the meat:
From what I get, Solid is about redecentralisation. That it, giving power back to the users.
As such, I am wondering, whether web analytics should be used.
I mean, to a certain degree it is about recording user behaviour, so the user should have a say in it.

Here in Europe we had the cookie directive and now GDPR. (Let me know if you need links).

So in theory the user should give informed consent before a web author is allowed to track. In praxis that happened rarely in that fashion.

So how would you design that with solid?

**Comment by Mitzi-Laszlo** [comment]
Great question @Ryuno-Ki, might be a conversation that would be suited to a forum thread?

**Comment by Ryuno-Ki** [comment]
Hi @Mitzi-Laszlo,

you are referring to https://forum.solidproject.org/ ? (Had to dig up the link from the node-solid-server project).
Opened a thread there: https://forum.solidproject.org/t/web-analytics-and-solid-principle/1573

Please keep this issue open so we can document an outcome here.

**Comment by Mitzi-Laszlo** [comment]
Any progress on wrapping this up?

**Comment by Ryuno-Ki** [comment]
Hey, it isn't even up for a week! (Only have 94 views according to Discourse).
First, people thought about GDPR only applying to companies. But it speaks of organisations. Which includes communities / forums and similiar. Since SoLiD is about Social (Linked Data), it would likely apply.

People would be willing to share data if it is for improving the service only.
Sadly, some applications are so dependent on tracking libraries to work, that they would fail also if you were to block the script.

In short: It's complicated.
(But that was too be expected).
